web/src/main/java/com/zebrunner/reporting/web/FileUtilController.java

```diff
@@ -34,7 +35,11 @@
 public class FileUtilController extends AbstractController implements FileUtilDocumentedController {
 
     private static final String DATA_FOLDER = "/opt/apk/%s";
-    private static final String[] ALLOWED_CONTENT_TYPES = {"image/png", "image/jpeg"};
+    private static final String[] ALLOWED_IMAGE_CONTENT_TYPES = {"image/png", "image/jpeg"};
+    private static final String[] APP_EXTENSIONS = {"app", "ipa", "apk"};
+
```

Comment:
Ok first of all you probably shouldn't put size into constant name - otherwise you'll have to rename it every time size restriction changes. I say stick to `MAX_IMAGE_SIZE` and `MAX_APP_PACKAGE_SIZE` accordingly. 
Second: there are megabytes and there are mebibytes. For the sake of clarity of your calculations, i would declare those values like following:
```java
private static final long MAX_IMAGE_SIZE = 2 * 1024 * 1024;
private static final long MAX_APP_PACKAGE_SIZE = 100 * 1024 * 1024;
```